nilearn/datasets/atlas.py

```diff
@@ -946,8 +946,8 @@ def fetch_coords_power_2011(legacy_format=True):
     """
     dataset_name = "power_2011"
     fdescr = get_dataset_descr(dataset_name)
-    package_directory = os.path.dirname(os.path.abspath(__file__))
-    csv = os.path.join(package_directory, "data", "power_2011.csv")
+    package_directory = Path(__file__).absolute().parent
```

Comment:
```
    package_directory = Path(__file__).absolute().parent
```

should we extract this in single function?

Reply:
Seems it is used in quite a few places.

Reply:
This is not clear to me.

Reply:
```
$ find nilearn -name '*.py' -exec grep -H "os.path.dirname(os.path.abspath(__file__))" {} \;

nilearn/datasets/atlas.py:    package_directory = os.path.dirname(os.path.abspath(__file__))
nilearn/datasets/atlas.py:    package_directory = os.path.dirname(os.path.abspath(__file__))
nilearn/datasets/atlas.py:    package_directory = os.path.dirname(os.path.abspath(__file__))
nilearn/datasets/struct.py:_package_directory = os.path.dirname(os.path.abspath(__file__))
nilearn/datasets/tests/test_func.py:currdir = os.path.dirname(os.path.abspath(__file__))
nilearn/datasets/tests/test_utils.py:currdir = os.path.dirname(os.path.abspath(__file__))
nilearn/datasets/func.py:    package_directory = os.path.dirname(os.path.abspath(__file__))
```

Between func, struct, atlas, we define this variable 5 times, so I was thinking of having a single function to do this, or a constant at the nilearn/datasets level.

Reply:
Do you think `nilearn/dataset/_util.py` would be a convenient place to define `PACKAGE_DIRECTORY` as a constant ?

Reply:
yeah this is where I would expect it